src/phrases/phrases-en.ts

```diff
@@ -1234,6 +1234,7 @@ const phrases = {
         resourceGroupPlaceholder: 'Grouping of resources',
         RGTooltip: 'Show resources Grouped or Ungrouped',
       },
+      expired: 'Expired',
```

Comment:
Du treng ikkje denne. Du kan bruke en variant av t(`subjectCategories.${subjectCategories.ARCHIVE_SUBJECTS}`) for det er den verdien som er satt i feltet.